index.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+require_once 'dbConnect.php';
+require_once 'functions.php';
+
+$db = connectToDb();
+
+$result = getReceipts($db);
+
+$receipts = displayData($result);
+
+if (isset($_POST["submit"])) {
+
+    $sName = testInput($_POST['supplier_name']);
+    $details = testInput($_POST['details']);
+    $amount = testInput($_POST['amount']);
+    $ccy = testInput($_POST['ccy']);
+    $date = testInput($_POST['date']);
+
+    insertData($sName,$details, $amount, $ccy, $date, $db);
+
+    header('Location: index.php');
+}
+?>
+<!DOCTYPE html>
+
+<html lang='en-gb'>
+
+<head>
+    <title>Collection App</title>
+    <meta name='viewport' content='width=device-width' />
+    <link rel='stylesheet' type='text/css' href='normalize.css'/>
+    <link rel='stylesheet' type='text/css' href='styles.css' />
+</head>
+
+<body>
+    <h1> Receipts </h1>
+    <div class='receiptsContainer'>
+        <?php echo $receipts;?>
+    </div>
+    <div class='box'>
+        <a class='button' href='#popup'>Add Receipt</a>
+    </div>
+
+    <div id='popup' class='overlay'>
+        <div class='popup'>
+            <h2> Add New Receipt </h2>
+            <a class='close' href='#'>&times;</a>
+            <div class='content'>
+                <form action='index.php' method='post'>
+                    <div class='form'>
```

Comment:
Again, more descriptive class name. '.form' would be confusing reading the css when it refers to a div